docs/reference/ingest.asciidoc

```diff
@@ -489,9 +489,10 @@ PUT _ingest/pipeline/my-pipeline
 ----
 
 [[template-snippets]]
-To access field values, enclose the field name in double curly brackets `{{ }}`
-to create a https://mustache.github.io[Mustache] template snippet. You can use
-template snippets to dynamically set field names.
+Several processor parameters support https://mustache.github.io[Mustache]
+template snippets. To access field values in a template snippet, enclose the
+field name in triple curly brackets:`{{{field-name}}}`. You can use template
+snippets to dynamically set field names.
```

Comment:
I debated noting whether the default variables (e.g., `{{field-name}}`) are HTML-escaped and that this syntax is
HTML-unescaped. But that seems more like documenting Mustache syntax rather than providing a quick, happy path that most users can pick up and move on with quickly.